pkg/volume/rbd/rbd_util.go

```diff
@@ -196,13 +196,14 @@ func getNbdDevFromImageAndPool(pool string, image string) (string, bool) {
 	return "", false
 }
 
+
 // Stat a path, if it doesn't exist, retry maxRetries times.
-func waitForPath(pool, image string, maxRetries int, useNbdDriver bool) (string, bool) {
+func waitForPath(pool, image string, maxRetries int, pathType string) (string, bool) {
```

Comment:
I prefer to keep this bool, since it looks like we only ever check for pathType == "nbd".

Reply:
Re: our gchat discussion - I'll either change it back to bool or move to an enum, if we use this code.